lib/data/external/AwsClient.js

```diff
@@ -7,6 +7,33 @@ const { prepareStream } = require('../../api/apiUtils/object/prepareStream');
 const { logHelper, removeQuotes, trimXMetaPrefix } = require('./utils');
 const { config } = require('../../Config');
 
+function _awsGetPreCheck(objectGetInfo, reqUids, callback) {
+    const log = createLogger(reqUids);
+    const key = typeof objectGetInfo === 'string' ? objectGetInfo :
```

Comment:
not needed for aws backend keys.  the old format is very old. it pre-dates our aws backend.